install.sh

```diff
@@ -2333,7 +1775,7 @@ ONETOKEN_SHARE_FILE="share/onetoken/onetoken.sh"
 
 FOLLOWER_CLEANUP_SHARE_FILE="share/hooks/raft/follower_cleanup"
 
-BACKUP_VMS_SHARE_FILE="share/scripts/backup_vms"
+PRE_CLEANUP_SHARE_FILE="share/pkgs/services/systemd/pre_cleanup"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

PRE_CLEANUP_SHARE_FILE appears unused. Verify use (or export if used externally).